web_src/css/markup/content.css

```diff
@@ -26,18 +26,22 @@
   float: left;
   padding-right: 4px;
   margin-left: -20px;
-  line-height: 1;
   color: inherit;
 }
 
 .markup .anchor .svg {
+  display: inline-block;
```

Comment:
IIRC there is a global style `.svg { inline-block }`, why it needs this?

Reply:
There is, but I thought to make markdown CSS stand on its own a bit more, but I guess I can revert this one.

Reply:
But on the other hand, it has more dependencies like colors, so it doesn't make much sense to duplicate this one.

Reply:
Removed.